lib/ae_page_objects/elements/collection.rb

```diff
@@ -77,7 +82,11 @@ def item_xpath
         #   .//div[contains(./text(), 'Example Text')]
         # See https://github.com/jnicklas/capybara#exactness for more information.
         #
-        query_args += [{:exact => true}] if query_args.first.to_sym == :xpath
+        if query_args.first.to_sym == :xpath
+          query_args.pop if query_args.last.is_a?(::Hash)
+          query_args.push options
```

Comment:
Is there any case where options might want to have `exact: false`? If so this overwrites the options.

Perhaps something like:

```
if query_args.last.is_a?(::Hash)
  query_args.last.update(options)
else
  query_args.push exact: true
end
```

I find this a little easier to read as well.